Samples/WindowsML/cpp-cmake/.gitignore

```diff
@@ -0,0 +1 @@
+__*
```

Comment:
I'm doing something wrong because this has no effect on a local clone & build.  I'm seeing all the build/* content as well as the __* dirs still listed.